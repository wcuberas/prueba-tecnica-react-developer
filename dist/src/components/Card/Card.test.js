import React from "react";
import Card from "./Card";
import { render } from "@testing-library/react";

const data = {
  title: "Ironman",
  imagen: 'https//:sdsdsd',
  description: "Bsdfsdfsdfsdfsdf",
  releaseYear: 2017,
};

it("matches snapshot", () => {
  const { asFragment } = render(<Card title={data.title} imagen={data.imagen} description={data.description} releaseYear={data.releaseYear} />);

  expect(asFragment()).toMatchSnapshot();
});