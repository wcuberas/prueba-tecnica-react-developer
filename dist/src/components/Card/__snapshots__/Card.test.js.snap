// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="containerCard"
  >
    <div
      class="card"
    >
      <div
        class="card__img"
        title="Ironman"
      >
        <img
          alt="Poster"
          src="https//:sdsdsd"
        />
      </div>
      <div
        class="card__title"
      >
        <button>
          <p>
            Ironman
          </p>
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
